crates/containerd-shim-wasm/src/sandbox/mod.rs

```diff
@@ -1,4 +1,45 @@
-//! Abstracts the sandboxing environment and execution context for a container.
+//! This module provides abstractions for managing WebAssembly containers in a sandboxed environment.
+//!
+//! This module gives you complete control over the container lifecycle and sandboxing,
+//! compared to the higher-level container module. It's useful when you need:
```

Comment:
It might be worth noting what you loose, in particular, precompilation and support for native linux containers (sidecars, etc).